### PR TITLE
docs(FrontEnd): Update README to include new syntax

### PR DESCRIPTION
**Pull request by jeffkwoh**
Update Front End handbook to include description for optional chaining and nullish coalescing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,13 +161,65 @@ Instead of using `mount()` from `enzyme` ...use this: `import {mountWithTheme} f
...
+
+### Nullish Coalescing
+
+This is a way to set a "default" value. e.g. previously you would do something like
+
+    let x = volume || 0.5;
+
+Which is a problem since `0` is a valid value for `volume`, but because it evaluates to `false` -y, we do not short circuit the expression and the value of `x` is `0.5`
+
+If instead we used null coalescing
```
Comment (jeffkwoh) [inline README.md]:
```suggestion
If instead we used [nullish coalescing](https://github.com/tc39/proposal-nullish-coalescing)
```

File: README.md [hunk 2]
```diff
@@ -161,13 +161,65 @@ Instead of using `mount()` from `enzyme` ...use this: `import {mountWithTheme} f
 We have decided to only use ECMAScript proposals that are in stage 3 (or later) (See [TC39 Proposals](https://github.com/tc39/proposals)). Additionally, because we are migrating to typescript, we will align with what their compiler supports.
 The only exception to this are decorators.
 
+## Optional Chaining and Null Coalescing
```
Comment (billyvg) [inline README.md]: Hmm maybe this header should be a bit more generic? Can we also add a link to the Quick Links suggestion at the top?

```suggestion
## New Syntax
```

Comment (jeffkwoh) [reply README.md]: Done.

**Review by billyvg — APPROVED** [review]
Can we add a PR title too?

**Pull request merged**